addons/test_mail/tests/test_message_post.py

```diff
@@ -823,6 +823,53 @@ def test_message_post_w_attachments(self):
                         ]
         )
 
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.addons.mail.models.mail_message_schedule')
+    def test_message_post_filter_main_attachments(self):
+        """
+        Test the message_main_attachment heuristics with an emphasis on the XML/PDF types.
+        -> we don't want XML files to be set as message_main_attachment
+        """
+        _attachments = [
+            ('List1', b'My xml attachment'),
+            ('List2', b'My pdf attachment'),
+        ]
+        _attachment_records = self.env['ir.attachment'].create(
```

Comment:
Seems only 2 are used ? no need to create 3 of them maybe ?

Reply:
Indeed!